### ISSUE TITLE
Cocoapods support?

### ISSUE THREAD

**Issue by dcgoss** [nicklockwood/SwipeView#150]
Would it be possible to integrate this repo with CocoaPods?

**Comment by nicklockwood** [comment]
I'm not sure what you mean? SwipeView is already on CocoaPods:

http://cocoapods.org/?q=swipeview

(It's the top one)

**Comment by dcgoss** [comment]
You're right! My fault. Perhaps adding this to the documentation (maybe in the "installation" section) would clear up some confusion?

**Comment by nicklockwood** [comment]
I figured if you're installing via Cocoapods then you'll already know what to do :-)

Fair point though.

**Comment by dcgoss** [comment]
I just didn't think to check whether it was available since it wasn't in the docs. It doesn't need to be a tutorial, just something like: "Hey, here's the podfile: `pod 'SwipeView'`" :)

**Comment by lewis-smith** [comment]
I agree, it's extremely common to include a line in the readme explaining how to install with pods.

**Comment by f33chobits** [comment]
Hi @dcgoss 
   Some repos just won't declare that in homepage, next time you can run 'pod search [repo-name]' which is 'pod search SwipeView' in this case.

**Comment by dcgoss** [comment]
Was having weird issues with forking - sorry for all the references.

**Comment by dcgoss** [comment]
Added a pull request for a quick addon to the README.

**Comment by TolgaB** [comment]
Still not added btw.
